fastkml/kml.py

```diff
@@ -958,8 +958,7 @@ class Document(_Container):
 
     def schemata(self):
         if self._schemata:
-            for schema in self._schemata:
-                yield schema
+            yield from self._schemata
```

Comment:
If support for python < 3.6 is dropped (as mentioned in #133), this can be accepted, otherwise Sourcery can be told to ignore it by changing the opening line of the method to `def schemata(self):  # sourcery skip: yield-from`